unittests/linuxliketests.py

```diff
@@ -1793,3 +1793,25 @@ def test_install_strip(self):
         self._run(install_cmd + ['--strip'], workdir=self.builddir)
         stdout = self._run(['file', '-b', lib])
         self.assertNotIn('not stripped', stdout)
+
+    @skip_if_not_base_option('b_static_analyzer')
```

Comment:
Does this belong in a different file (not `linuxliketests.py`). I'll let someone smarter than me about test organization comment on that.